db/migrate/20230607161220_create_users.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.text :photo
```

Comment:
- [OPTIONAL] it will be much better for the logo to use string data type instead of text, It's important to consider the potential size of the data you need to store when deciding between the string and text data types. If the data is relatively short and fits within the limits of a string data type (usually up to 255 characters), using string can provide better performance and efficiency. However, for larger amounts of text like description, biography, ...., the text data type is the appropriate choice.